vagrant/base.sh

```diff
@@ -17,23 +17,33 @@
 
 set -e
 
-if [ -z `which javac` ]; then
-    apt-get -y update
-    apt-get install -y software-properties-common python-software-properties
-    add-apt-repository -y ppa:webupd8team/java
-    apt-get -y update
-
-    # Try to share cache. See Vagrantfile for details
-    mkdir -p /var/cache/oracle-jdk7-installer
-    if [ -e "/tmp/oracle-jdk7-installer-cache/" ]; then
-        find /tmp/oracle-jdk7-installer-cache/ -not -empty -exec cp '{}' /var/cache/oracle-jdk7-installer/ \;
+if [ $(cat /etc/system-release | grep -oE 'Fedora release [[:digit:]]+') == 'Fedora release 21' ]
```

Comment:
This is complaining about `bash: [: too many arguments`. It looks like this requires using `[[ ]]` for the test rather than just `[ ]`. Splitting into multiple lines (i.e. assigning the results of the `cat` + `grep` to a var and then doing the comparison) might also help fix this.

Also, should we really be testing specifically for `Fedora release 21`? Would a test for `which yum` work?